frontend/src/routes/+layout.ts

```diff
@@ -1,10 +1,10 @@
 import type { main_Country } from "$lib/generated/go"
 import type { LayoutLoad } from "./$types"
-import { GO_API } from "$lib/variables"
+import { PYTHON_API } from "$lib/variables"
 
 export const load: LayoutLoad = async ({ fetch }) => {
   const fetchCountries = async (): Promise<main_Country[]> => {
```

Comment:
We should probably run the type generator again + change this to the Python-exported types

Reply:
Generated type should now be used